insert_food.py

```diff
@@ -2,8 +2,8 @@
 from psql_connetion import psql_connection
 
 def insert_values(food_name, food_calories):
-    connection, crsr = psql_connection("food_database.ini", "postgresql")
-    last_id_number = crsr.execute("SELECT max(food_id) FROM food_calories")
+    connection, crsr = psql_connection("database.ini", "postgresql")
+    crsr.execute("SELECT max(food_id) FROM food_calories")
```

Comment:
Put the connection to the database in a different function, because you'll need it not only in this function but in plenty of others (lke when you will need to read from the BD, update, etc)